776 pbinfo/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+
+using namespace std;
+
+/*Se dă o matrice cu n linii şi m coloane şi elemente numere naturale. Să se determine câte linii ale matricei au toate elementele egale.
+
+date de intrare:
+4 6
+23 23 15 23 18 23
+17 17 17 17 17 17
+17 18 13 18 12 15
+34 34 34 34 34 34
+
+date dde iesire:
+2
+*/
+
```

Comment:
poti face logica in timp ce citesti datele, nu ai nevoie de inca doua foruri pentu asta